polkadot-archive/src/main.rs

```diff
@@ -20,24 +20,27 @@ use log::{warn, error};
 use failure::Error;
 // use substrate_archive::prelude::*;
 use serde::{Deserialize, Serialize};
+use serde_json::{json, Value};
 use substrate_archive::{
     Archive, System, Module, RawExtrinsic,
     OldExtrinsic, ToDatabaseExtrinsic,
     ExtractCall, SrmlExt, NotHandled,
```

Comment:
should `SrmlExt` be renamed at some point to `PaintExt`

Reply:
Eventually, Pallet/Pallette now I guess